setup.py

```diff
@@ -53,7 +53,8 @@ def get_readme_content():
             'click>=7,<8',
             'docker>=3,<5',
             'pyyaml>=3,<6',
-            'requests>=2,<3',
+            'requests>=2,<2.29.0',  # https://github.com/docker/docker-py/issues/3113
+            'urllib3<2.0',          # https://github.com/docker/docker-py/issues/3113
```

Comment:
This should not be necessary as `requests < 2.29`  already requires `urllib3 < 1.27` ([here](https://github.com/psf/requests/blob/v2.28.2/setup.py#L64))

Reply:
Removed in 20004e86f06a4625fb327d1ec4b908f49e1b14d1.